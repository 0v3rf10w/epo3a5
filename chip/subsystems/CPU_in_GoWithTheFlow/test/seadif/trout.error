This is the file 'seadif/trout.error' which was created by
the program 'trout' on Mon Jan  5 11:42:22 2015
Trout was verifying the netlist description of
your circuit circuit 'cpu' with the current layout in seadali.
The following nets in the layout are not according to
the netlist description of circuit 'cpu'. They are also
marked in the layout.
------------------------- list of errors -------------------------
##### suspected short-circuit no. 1:
  > net 'cpu_dec_o_2_port' (1 terms, e.g. terminal  'decoder_argout_7_0_2')
  > net 'cpu_dec_bregld_2_port' (2 terms, e.g. terminal 'decoder_bregld_4_0_2')
  > net 'cpu_dec_bbufoe_2_port' (2 terms, e.g. terminal 'decoder_bbufoe_4_0_2')
  > net 'cpu_instr_11_0_2' (3 terms, e.g. terminal 'decoder_in_11_0_2')
##### suspected short-circuit no. 2:
  > net 'cpu_dec_bregld_4_port' (1 terms, e.g. terminal  'A')
  > net 'cpu_dec_abufoe' (2 terms, e.g. terminal 'decoder_abufoe')
##### suspected short-circuit no. 3:
  > net 'cpu_dec_o_6_port' (1 terms, e.g. terminal  'decoder_argout_7_0_6')
  > net 'cpu_instr_11_0_6' (3 terms, e.g. terminal 'decoder_in_11_0_6')
##### suspected short-circuit no. 4:
  > net 'cpu_dec_o_5_port' (1 terms, e.g. terminal  'decoder_argout_7_0_5')
  > net 'cpu_instr_11_0_5' (3 terms, e.g. terminal 'decoder_in_11_0_5')
##### suspected short-circuit no. 5:
  > net 'cpu_dec_o_1_port' (1 terms, e.g. terminal  'decoder_argout_7_0_1')
  > net 'cpu_dec_bregld_1_port' (2 terms, e.g. terminal 'reg_select_4_0_1')
  > net 'cpu_dec_bbufoe_1_port' (2 terms, e.g. terminal 'decoder_bbufoe_4_0_1')
  > net 'cpu_instr_11_0_1' (3 terms, e.g. terminal 'decoder_in_11_0_1')
##### suspected short-circuit no. 6:
  > net 'cpu_dec_o_4_port' (1 terms, e.g. terminal  'decoder_argout_7_0_4')
  > net 'cpu_instr_11_0_4' (3 terms, e.g. terminal 'decoder_in_11_0_4')
##### suspected short-circuit no. 7:
  > net 'cpu_dec_o_3_port' (1 terms, e.g. terminal  'decoder_argout_7_0_3')
  > net 'cpu_dec_bregld_3_port' (2 terms, e.g. terminal 'reg_select_4_0_3')
  > net 'cpu_dec_bbufoe_3_port' (2 terms, e.g. terminal 'decoder_bbufoe_4_0_3')
  > net 'cpu_instr_11_0_3' (3 terms, e.g. terminal 'decoder_in_11_0_3')
##### suspected short-circuit no. 8:
  > net 'cpu_dec_o_7_port' (1 terms, e.g. terminal  'decoder_argout_7_0_7')
  > net 'cpu_instr_11_0_7' (3 terms, e.g. terminal 'decoder_in_11_0_7')
##### suspected short-circuit no. 9:
  > net 'cpu_dec_o_0_port' (1 terms, e.g. terminal  'decoder_argout_7_0_0')
  > net 'cpu_dec_bregld_0_port' (2 terms, e.g. terminal 'reg_select_4_0_0')
  > net 'cpu_dec_bbufoe_0_port' (2 terms, e.g. terminal 'decoder_bbufoe_4_0_0')
  > net 'cpu_instr_11_0_0' (3 terms, e.g. terminal 'decoder_in_11_0_0')

WARNING: 9 short-circuit(s) were detected among the 105 nets.
         These nets were separatedly specified in the circuit
         but are connected in the layout.  This may occur because of
         feed-throughs in sub-cells and it doesn't necessarily
         imply an error.
         In the layout each short-circuit is indicated by a marker
         which indicates a position on the pattern: <=-SHORT_<no>
------------------------------------------------------------------
lock
