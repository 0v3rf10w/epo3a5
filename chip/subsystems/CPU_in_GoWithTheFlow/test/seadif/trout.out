>>>>>> trout: automatic router for Sea-of-Gates           <<<<<<
>>>>>> (c) 1995 Patrick Groeneveld, Delft Univ. of Techn. <<<<<<
------ reading image description file '/data/public/common/software/nelsis/5.4.1.1/share/lib/celllibs/fishbone/image.seadif' ------
WARNING: Seadif file "/home/bmeekes/CPU/test/seadif/sealib.sdf" is not a magic cookie (skipped).

------ reading placement 'Tmp_Cell_(cpu(cpu(test)))' ------
------ building data structure  ------
------ making grid  ------
------ routing started Mon Jan  5 11:42:02 2015
 105 nets have to be routed.
Routing net 'cpu_out_7_0_3' (2/2): .b OK!
Routing net 'cpu_out_7_0_0' (2/2): .b OK!
Routing net 'cpu_out_7_0_2' (2/2): .b OK!
Routing net 'cpu_en' (2/2): .b OK!
Routing net 'cpu_pc_7_0_7' (2/2): .b OK!
Routing net 'cpu_pc_7_0_6' (2/2): .b OK!
Routing net 'cpu_pc_7_0_5' (2/2): .b OK!
Routing net 'cpu_pc_7_0_4' (2/2): .b OK!
Routing net 'cpu_pc_7_0_3' (2/2): .b OK!
Routing net 'cpu_pc_7_0_2' (2/2): .b OK!
Routing net 'cpu_pc_7_0_1' (2/2): .b OK!
Routing net 'cpu_pc_7_0_0' (2/2): .b OK!
Routing net 'cpu_in_7_0_7' (2/2): .b OK!
Routing net 'cpu_in_7_0_6' (2/2): .b OK!
Routing net 'cpu_in_7_0_5' (2/2): .b OK!
Routing net 'cpu_in_7_0_4' (2/2): .b OK!
Routing net 'cpu_in_7_0_3' (2/2): .b OK!
Routing net 'cpu_in_7_0_2' (2/2): .b OK!
Routing net 'cpu_in_7_0_1' (2/2): .b OK!
Routing net 'cpu_in_7_0_0' (2/2): .b OK!
Routing net 'cpu_out_7_0_7' (2/2): .b OK!
Routing net 'cpu_out_7_0_6' (2/2): .b OK!
Routing net 'cpu_out_7_0_5' (2/2): .b OK!
Routing net 'cpu_out_7_0_4' (2/2): .b OK!
Routing net 'cpu_out_7_0_1' (2/2): .b OK!
Routing net 'cpu_dec_bbufoe_4_port' (2/2): . OK!
Routing net 'cpu_alu_out_1_port' (2/2): . OK!
Routing net 'cpu_alu_out_4_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_4_port' (2/2): . OK!
Routing net 'cpu_test_inc' (3/3): ..b OK!
Routing net 'cpu_dec_abufoe' (2/2): . OK!
Routing net 'n8' (2/2): . OK!
Routing net 'n2' (2/2): . OK!
Routing net 'cpu_alu_out_3_port' (2/2): . OK!
Routing net 'cpu_dec_o_6_port' (2/2): . OK!
Routing net 'cpu_dec_o_5_port' (2/2): . OK!
Routing net 'cpu_dec_o_1_port' (2/2): . OK!
Routing net 'cpu_dec_o_2_port' (2/2): . OK!
Routing net 'cpu_areg_out_4_port' (3/3): .. OK!
Routing net 'cpu_areg_out_3_port' (3/3): .. OK!
Routing net 'cpu_dec_o_4_port' (2/2): . OK!
Routing net 'cpu_dec_o_3_port' (2/2): . OK!
Routing net 'cpu_dec_o_7_port' (2/2): . OK!
Routing net 'cpu_test_ld' (3/3): ..b OK!
Routing net 'cpu_dec_bregld_3_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_1_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_2_port' (2/2): . OK!
Routing net 'cpu_dec_o_0_port' (2/2): . OK!
Routing net 'cpu_alu_out_7_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_0_port' (2/2): . OK!
Routing net 'cpu_areg_out_0_port' (3/3): .. OK!
Routing net 'n9' (2/2): . OK!
Routing net 'n4' (2/2): . OK!
Routing net 'n5' (2/2): . OK!
Routing net 'cpu_dec_bbufoe_2_port' (2/2): . OK!
Routing net 'cpu_areg_out_1_port' (3/3): .. OK!
Routing net 'cpu_alu_out_2_port' (2/2): . OK!
Routing net 'cpu_dec_ibufoe' (2/2): . OK!
Routing net 'cpu_dec_bbufoe_1_port' (2/2): . OK!
Routing net 'n7' (2/2): . OK!
Routing net 'cpu_areg_out_5_port' (3/3): .. OK!
Routing net 'cpu_dec_bbufoe_0_port' (2/2): . OK!
Routing net 'cpu_alu_out_6_port' (2/2): . OK!
Routing net 'n6' (2/2): . OK!
Routing net 'cpu_alu_out_0_port' (2/2): . OK!
Routing net 'cpu_areg_out_7_port' (3/3): .. OK!
Routing net 'cpu_alu_out_5_port' (2/2): . OK!
Routing net 'cpu_areg_out_2_port' (3/3): .. OK!
Routing net 'cpu_areg_out_6_port' (3/3): .. OK!
Routing net 'n12' (2/2): . OK!
Routing net 'n1' (2/2): . OK!
Routing net 'cpu_dec_bbufoe_3_port' (2/2): . OK!
Routing net 'n11' (2/2): . OK!
Routing net 'n3' (2/2): . OK!
Routing net 'n10' (2/2): . OK!
Routing net 'cpu_alu_op_2_port' (2/2): x FAILED
Routing net 'cpu_dec_aregld' (2/2): . OK!
Routing net 'cpu_alu_op_0_port' (2/2): . OK!
Routing net 'cpu_alu_op_1_port' (2/2): . OK!
Routing net 'cpu_alu_c' (2/2): . OK!
Routing net 'cpu_test_data_7_0_7' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_4' (7/7): ......b OK!
Routing net 'cpu_instr_11_0_3' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_1' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_10' (3/3): ..b OK!
Routing net 'cpu_alu_z' (2/2): . OK!
Routing net 'cpu_instr_11_0_7' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_6' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_2' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_5' (3/3): ..b OK!
Routing net 'cpu_rst' (4/4): ...b OK!
Routing net 'cpu_instr_11_0_0' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_4' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_8' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_11' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_9' (3/3): ..b OK!
Routing net 'cpu_test_data_7_0_0' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_2' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_3' (7/7): ......b OK!
Routing net 'cpu_clk' (5/5): ....b OK!
Routing net 'vss' : . (Power special)
Routing net 'cpu_test_data_7_0_5' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_6' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_1' (7/7): ......b OK!
--> 99.05 % elapsed:   6.3 sec. expect another   0.4 sec. <--
Routing net 'vdd' :  (Power special)
----- Don't worry, I'm trying again -----
Routing net 'cpu_out_7_0_0' (2/2): .b OK!
Routing net 'cpu_out_7_0_3' (2/2): .b OK!
Routing net 'cpu_out_7_0_2' (2/2): .b OK!
Routing net 'cpu_out_7_0_1' (2/2): .b OK!
Routing net 'cpu_out_7_0_4' (2/2): .b OK!
Routing net 'cpu_out_7_0_5' (2/2): .b OK!
Routing net 'cpu_out_7_0_6' (2/2): .b OK!
Routing net 'cpu_out_7_0_7' (2/2): .b OK!
Routing net 'cpu_in_7_0_0' (2/2): .b OK!
Routing net 'cpu_in_7_0_1' (2/2): .b OK!
Routing net 'cpu_in_7_0_2' (2/2): .b OK!
Routing net 'cpu_in_7_0_3' (2/2): .b OK!
Routing net 'cpu_in_7_0_4' (2/2): .b OK!
Routing net 'cpu_in_7_0_5' (2/2): .b OK!
Routing net 'cpu_in_7_0_6' (2/2): .b OK!
Routing net 'cpu_in_7_0_7' (2/2): .b OK!
Routing net 'cpu_pc_7_0_0' (2/2): .b OK!
Routing net 'cpu_pc_7_0_1' (2/2): .b OK!
Routing net 'cpu_pc_7_0_2' (2/2): .b OK!
Routing net 'cpu_pc_7_0_3' (2/2): .b OK!
Routing net 'cpu_pc_7_0_4' (2/2): .b OK!
Routing net 'cpu_pc_7_0_5' (2/2): .b OK!
Routing net 'cpu_pc_7_0_6' (2/2): .b OK!
Routing net 'cpu_pc_7_0_7' (2/2): .b OK!
Routing net 'cpu_en' (2/2): .b OK!
Routing net 'cpu_dec_bbufoe_4_port' (2/2): . OK!
Routing net 'cpu_alu_op_2_port' (2/2): . OK!
Routing net 'cpu_alu_out_1_port' (2/2): . OK!
Routing net 'cpu_alu_out_4_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_4_port' (2/2): . OK!
Routing net 'cpu_test_inc' (3/3): ..b OK!
Routing net 'cpu_dec_abufoe' (2/2): . OK!
Routing net 'n8' (2/2): . OK!
Routing net 'n2' (2/2): . OK!
Routing net 'cpu_alu_out_3_port' (2/2): . OK!
Routing net 'cpu_dec_o_6_port' (2/2): . OK!
Routing net 'cpu_dec_o_5_port' (2/2): . OK!
Routing net 'cpu_dec_o_1_port' (2/2): . OK!
Routing net 'cpu_dec_o_2_port' (2/2): x FAILED
Routing net 'cpu_areg_out_4_port' (3/3): .. OK!
Routing net 'cpu_areg_out_3_port' (3/3): .. OK!
Routing net 'cpu_dec_o_4_port' (2/2): . OK!
Routing net 'cpu_dec_o_3_port' (2/2): . OK!
Routing net 'cpu_dec_o_7_port' (2/2): . OK!
Routing net 'cpu_test_ld' (3/3): ..b OK!
Routing net 'cpu_dec_bregld_1_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_3_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_2_port' (2/2): . OK!
Routing net 'cpu_dec_o_0_port' (2/2): . OK!
Routing net 'cpu_alu_out_7_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_0_port' (2/2): . OK!
Routing net 'n9' (2/2): . OK!
Routing net 'cpu_areg_out_0_port' (3/3): .. OK!
Routing net 'n5' (2/2): . OK!
Routing net 'n4' (2/2): . OK!
Routing net 'cpu_areg_out_1_port' (3/3): .. OK!
Routing net 'cpu_dec_bbufoe_2_port' (2/2): . OK!
Routing net 'cpu_alu_out_2_port' (2/2): . OK!
Routing net 'cpu_dec_ibufoe' (2/2): . OK!
Routing net 'n7' (2/2): . OK!
Routing net 'cpu_dec_bbufoe_1_port' (2/2): . OK!
Routing net 'cpu_areg_out_5_port' (3/3): .. OK!
Routing net 'cpu_dec_bbufoe_0_port' (2/2): . OK!
Routing net 'cpu_alu_out_6_port' (2/2): . OK!
Routing net 'n6' (2/2): . OK!
Routing net 'cpu_alu_out_0_port' (2/2): . OK!
Routing net 'cpu_areg_out_7_port' (3/3): .. OK!
Routing net 'cpu_alu_out_5_port' (2/2): . OK!
Routing net 'cpu_areg_out_2_port' (3/3): .. OK!
Routing net 'cpu_areg_out_6_port' (3/3): .. OK!
Routing net 'n12' (2/2): . OK!
Routing net 'n1' (2/2): . OK!
Routing net 'cpu_dec_bbufoe_3_port' (2/2): . OK!
Routing net 'n11' (2/2): . OK!
Routing net 'n3' (2/2): . OK!
Routing net 'n10' (2/2): . OK!
Routing net 'cpu_dec_aregld' (2/2): . OK!
Routing net 'cpu_alu_op_0_port' (2/2): . OK!
Routing net 'cpu_alu_op_1_port' (2/2): . OK!
Routing net 'cpu_alu_c' (2/2): . OK!
Routing net 'cpu_test_data_7_0_7' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_4' (7/7): ......b OK!
Routing net 'cpu_instr_11_0_3' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_1' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_10' (3/3): ..b OK!
Routing net 'cpu_alu_z' (2/2): . OK!
Routing net 'cpu_instr_11_0_7' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_6' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_2' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_5' (3/3): ..b OK!
Routing net 'cpu_rst' (4/4): ...b OK!
Routing net 'cpu_instr_11_0_0' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_4' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_8' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_11' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_9' (3/3): ..b OK!
Routing net 'cpu_test_data_7_0_0' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_2' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_3' (7/7): ......b OK!
Routing net 'cpu_clk' (5/5): ....b OK!
Routing net 'vss' : . (Power special)
Routing net 'cpu_test_data_7_0_5' (7/7): ......b OK!
--> 97.14 % elapsed:   6.0 sec. expect another   0.4 sec. <--
Routing net 'cpu_test_data_7_0_6' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_1' (7/7): ......b OK!
Routing net 'vdd' :  (Power special)
----- Don't worry, I'm trying again -----
Routing net 'cpu_out_7_0_3' (2/2): .b OK!
Routing net 'cpu_out_7_0_0' (2/2): .b OK!
Routing net 'cpu_out_7_0_2' (2/2): .b OK!
Routing net 'cpu_dec_o_2_port' (2/2): . OK!
Routing net 'cpu_en' (2/2): .b OK!
Routing net 'cpu_pc_7_0_7' (2/2): .b OK!
Routing net 'cpu_pc_7_0_6' (2/2): .b OK!
Routing net 'cpu_pc_7_0_5' (2/2): .b OK!
Routing net 'cpu_pc_7_0_4' (2/2): .b OK!
Routing net 'cpu_pc_7_0_3' (2/2): .b OK!
Routing net 'cpu_pc_7_0_2' (2/2): .b OK!
Routing net 'cpu_pc_7_0_1' (2/2): .b OK!
Routing net 'cpu_pc_7_0_0' (2/2): .b OK!
Routing net 'cpu_in_7_0_7' (2/2): .b OK!
Routing net 'cpu_in_7_0_6' (2/2): .b OK!
Routing net 'cpu_in_7_0_5' (2/2): .b OK!
Routing net 'cpu_in_7_0_4' (2/2): .b OK!
Routing net 'cpu_in_7_0_3' (2/2): .b OK!
Routing net 'cpu_in_7_0_2' (2/2): .b OK!
Routing net 'cpu_in_7_0_1' (2/2): .b OK!
Routing net 'cpu_in_7_0_0' (2/2): .b OK!
Routing net 'cpu_out_7_0_7' (2/2): .b OK!
Routing net 'cpu_out_7_0_6' (2/2): .b OK!
Routing net 'cpu_out_7_0_5' (2/2): .b OK!
Routing net 'cpu_out_7_0_4' (2/2): .b OK!
Routing net 'cpu_out_7_0_1' (2/2): .b OK!
Routing net 'cpu_dec_bbufoe_4_port' (2/2): . OK!
Routing net 'cpu_alu_op_2_port' (2/2): . OK!
Routing net 'cpu_alu_out_1_port' (2/2): . OK!
Routing net 'cpu_alu_out_4_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_4_port' (2/2): . OK!
Routing net 'cpu_test_inc' (3/3): ..b OK!
Routing net 'cpu_dec_abufoe' (2/2): . OK!
Routing net 'n8' (2/2): . OK!
Routing net 'n2' (2/2): . OK!
Routing net 'cpu_alu_out_3_port' (2/2): . OK!
Routing net 'cpu_dec_o_6_port' (2/2): . OK!
Routing net 'cpu_dec_o_5_port' (2/2): . OK!
Routing net 'cpu_dec_o_1_port' (2/2): . OK!
Routing net 'cpu_areg_out_4_port' (3/3): .. OK!
Routing net 'cpu_areg_out_3_port' (3/3): .. OK!
Routing net 'cpu_dec_o_4_port' (2/2): . OK!
Routing net 'cpu_dec_o_3_port' (2/2): . OK!
Routing net 'cpu_dec_o_7_port' (2/2): . OK!
Routing net 'cpu_test_ld' (3/3): ..b OK!
Routing net 'cpu_dec_bregld_3_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_1_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_2_port' (2/2): . OK!
Routing net 'cpu_dec_o_0_port' (2/2): . OK!
Routing net 'cpu_alu_out_7_port' (2/2): . OK!
Routing net 'cpu_dec_bregld_0_port' (2/2): . OK!
Routing net 'cpu_areg_out_0_port' (3/3): .. OK!
Routing net 'n9' (2/2): . OK!
Routing net 'n4' (2/2): . OK!
Routing net 'n5' (2/2): . OK!
Routing net 'cpu_dec_bbufoe_2_port' (2/2): . OK!
Routing net 'cpu_areg_out_1_port' (3/3): .. OK!
Routing net 'cpu_alu_out_2_port' (2/2): . OK!
Routing net 'cpu_dec_ibufoe' (2/2): . OK!
Routing net 'cpu_dec_bbufoe_1_port' (2/2): . OK!
Routing net 'n7' (2/2): . OK!
Routing net 'cpu_areg_out_5_port' (3/3): .. OK!
Routing net 'cpu_dec_bbufoe_0_port' (2/2): . OK!
Routing net 'cpu_alu_out_6_port' (2/2): . OK!
Routing net 'n6' (2/2): . OK!
Routing net 'cpu_alu_out_0_port' (2/2): . OK!
Routing net 'cpu_areg_out_7_port' (3/3): .. OK!
Routing net 'cpu_alu_out_5_port' (2/2): . OK!
Routing net 'cpu_areg_out_2_port' (3/3): .. OK!
Routing net 'cpu_areg_out_6_port' (3/3): .. OK!
Routing net 'n12' (2/2): . OK!
Routing net 'n1' (2/2): . OK!
Routing net 'cpu_dec_bbufoe_3_port' (2/2): . OK!
Routing net 'n11' (2/2): . OK!
Routing net 'n3' (2/2): . OK!
Routing net 'n10' (2/2): . OK!
Routing net 'cpu_dec_aregld' (2/2): . OK!
Routing net 'cpu_alu_op_0_port' (2/2): . OK!
Routing net 'cpu_alu_op_1_port' (2/2): . OK!
Routing net 'cpu_alu_c' (2/2): . OK!
Routing net 'cpu_test_data_7_0_7' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_4' (7/7): ......b OK!
Routing net 'cpu_instr_11_0_3' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_1' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_10' (3/3): ..b OK!
Routing net 'cpu_alu_z' (2/2): . OK!
Routing net 'cpu_instr_11_0_7' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_6' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_2' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_5' (3/3): ..b OK!
Routing net 'cpu_rst' (4/4): ...b OK!
Routing net 'cpu_instr_11_0_0' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_4' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_8' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_11' (3/3): ..b OK!
Routing net 'cpu_instr_11_0_9' (3/3): ..b OK!
Routing net 'cpu_test_data_7_0_0' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_2' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_3' (7/7): ......b OK!
Routing net 'cpu_clk' (5/5): ....b OK!
Routing net 'vss' : . (Power special)
Routing net 'cpu_test_data_7_0_5' (7/7): ......b OK!
--> 97.14 % elapsed:   6.0 sec. expect another   0.4 sec. <--
Routing net 'cpu_test_data_7_0_6' (7/7): ......b OK!
Routing net 'cpu_test_data_7_0_1' (7/7): ......b OK!
Routing net 'vdd' :  (Power special)
READY
----- Some interesting statistics ------
Total number of nets:                 105 (100% completion)
Total length of routed wires:         31342 grids = 275.810 mm
No. of transistors (total / used):    41064/8738 = 21.28 %
CPU-time consumption:                18.95 sec.
Elapsed time during routing:         19.13 sec. (99.11 % of cpu)
------ Verifying Connectivity ------
WARNING: 9 short-circuit(s) were detected among the 105 nets.
------ Writing 'Tmp_Cell_(cpu(cpu(test)))' ------
------ Trout: task completed on Mon Jan  5 11:42:23 2015
 